src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -238,6 +262,10 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
 
   return (
     <>
+      {/* dotcom and dotcomConfig need to be setup before the main dotcom javascript file is loaded */}
+      {adsEnabled && showAdsBasedOnLocation && !isAmp && (
```

Comment:
Please can we have _one_ `isAdsEnabled` Boolean? If we're required to check toggles, we can that to the array on L151.

Reply:
@simonsinclair it's tricky because `adsEnabled` is checked in the `Ads` container but not in `CanonicalAdBootstrapJs`, so if we add that check there it might seem like it's done twice, we can always move the toggle to `CanonicalAdBootstrapJs`

Reply:
Sorry, I don't think I explained myself very well. Can we use `isAdsEnabled` for _all_ pre-render checks and add the `isAdsToggleEnabled` (currently `adsEnabled`) to the array on L151?

Reply:
done.